OutSystems/outsystems-datagrid-reactive
# ROU-3121: Creating SetColumnVisibility API method

This PR creates the Column Picker's SetColumnVisibility API method, which is responsible for hiding/showing visible columns on the column picker.

### Test Steps
1. Open Column Picker
2. Product Name and Product Category should show on column picker.
3. Toggle ShowHiddenColumns to false and Press ShowHiddenColumns button.
4. Open Column Picker
5. Product Name and Product Category should not show on column picker.
